### DISCUSSION TITLE
how to create anchor based AR in model-viewer like WebXR "Hit Test"

### DISCUSSION THREAD

**Discussion by saurabhburadkar** [google/model-viewer#4272] [General]
I want to create AR like Hit test of webxr in model viewer. how to create this?
test link :- https://immersive-web.github.io/webxr-samples/hit-test.html

**Comment by elalish** [comment]
I'm not quite sure what you mean - model-viewer's webXR mode already uses hit test to place and move the model. What do you want to do with hit test?

**Comment by saurabhburadkar** [comment]
I want to create the same experience as a WebXR hit test, where the user clicks on the surface, the model will augment there. It is possible in the model viewer?

**Reply by elalish** [reply]
You mean, as opposed to our auto-placement and dragging to reposition? You'd need to fork our library to change our UX at that level, or propose a PR with some improvement.

**Reply by saurabhburadkar** [reply]
okay, Got it, Thanks for the response
